doc/source/examples/dem/granular-mixer/granular-mixer.rst

```diff
@@ -0,0 +1,144 @@
+                set friction coefficient particles      = 0.5
+                set rolling friction particles          = 0.2
+        	end
+        	subsection particle type 1
+                set size distribution type              = uniform
+                set diameter                            = 0.0015
+                set number                              = 23500
+                set density particles                   = 1500
+                set young modulus particles             = 1000000
+                set poisson ratio particles             = 0.5
+                set restitution coefficient particles   = 0.5
+                set friction coefficient particles      = 0.5
+                set rolling friction particles          = 0.2
+        	end
+                set young modulus wall                  = 1000000
+                set poisson ratio wall                  = 0.5
+                set restitution coefficient wall        = 0.5
+                set friction coefficient wall           = 0.5
+                set rolling friction wall               = 0.2
+    end
+
+
+Solid objects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this subsection, the floating meshes are defined. We can use dealii or gmsh to create the floating meshes. At the moment, solid objects (floating meshes) in Lethe have to be defined using triangular (simplex) meshes. Only triangular 2D meshes of 3D surfaces in the dem_3d solver are presently supported. Quadrilateral 2D meshes of 3D surfaces and 1D mesh of 2D surfaces are not supported at the moment. For each floating mesh, we need to specify a ``translational velocity``, a ``angular velocity``, and a ``center of rotation``. In this example, we only need an angular motion of the impeller. Note that the ``center of rotation`` of the impeller is at 0, 0, 0.
+
+.. code-block:: text
+
+    subsection solid objects
+        set number of solids        = 1
+        subsection  solid object 0
+        subsection mesh
+            set type                = gmsh
+            set file name           = pitched_blade_impeller.msh
+            set simplex		       = true
+            set initial refinement  = 0
+        end
+    	 
+        subsection translational velocity
+            set Function expression = 0 ; 0 ; 0
+        end
+
+        subsection angular velocity
+            set Function expression = if(t > 0.5 , 6 ,0) ; 0 ; 0
+        end
+    
+        subsection center of rotation
+            # X COR
+            set x				= 0
+            # Y COR
+            set y				= 0
+            # Z COR
+            set z				= 0
+        end
+    
+      end
+    
+    end
+
+
+Running the simulation
+----------------------
+This simulation can be launched by (in parallel mode on 16 processes):
+
+.. code-block:: text
+
+  mpirun -np 16 dem_3d granular_mixer.prm
+
+.. warning::
+	This example takes approximately 30 hours on 16 cores.
+
+
+Results
+---------
+
+Animation of the granular mixing simulation:
```

Comment:
I would add a possibility for extension section where you could suggest that the mixing could be monitored using the distribution of the particle type. Otherwise that's a cool nice and simple example.

Reply:
Added.